File: 1/1-20.hs
gcd' :: Int -> Int -> Int
gcd' a 0 = a
gcd' a b = gcd' b (a `mod` b)

main :: IO ()
main = do
    let result = gcd' 206 40
    putStrLn $ "GCD(206, 40) = " ++ show result

File: 1/1-20.ts
function gcd(a: number, b: number): number {
    // Computes the greatest common divisor of a and b using Euclid's algorithm.
    if (b === 0) {
        return a;
    } else {
        return gcd(b, a % b);
    }
}

const result = gcd(206, 40);
console.log(`GCD(206, 40) = ${result}`);